### PR TITLE
docs(spanner): fix single character parameter

### LINKED ISSUE #47: render bug in python API documentation
https://googleapis.dev/python/spanner/latest/transaction-api.html  

The Parameters column is only 1 character wide

screenshot

### PR DESCRIPTION
**Pull request by HemangChothani**
Fixes #47 

**Transaction class:**

Screenshot from 2020-04-07 20-20-20

**Snapshot class**
Screenshot from 2020-04-07 20-21-43

### CODE REVIEW

**Comment by HemangChothani** [comment]
@larkee  I have tried something to fix that, if it looks good to you than i will take cleanup for whole library.

File: google/cloud/spanner_v1/snapshot.py [hunk 1]
```diff
@@ -178,12 +178,13 @@ def execute_sql(
             required if parameters are passed.
 
         :type query_mode:
-            :class:`google.cloud.spanner_v1.proto.ExecuteSqlRequest.QueryMode`
-        :param query_mode: Mode governing return of results / query plan. See
-            https://cloud.google.com/spanner/reference/rpc/google.spanner.v1#google.spanner.v1.ExecuteSqlRequest.QueryMode1
+            :class:`~google.cloud.spanner_v1.proto.ExecuteSqlRequest.QueryMode`
+        :param query_mode: Mode governing return of results / query plan.
+            See :
+            `QueryMode1 <https://cloud.google.com/spanner/reference/rpc/google.spanner.v1#google.spanner.v1.ExecuteSqlRequest.QueryMode1>`_
```
Comment (larkee) [inline google/cloud/spanner_v1/snapshot.py:184]: Replace space with full stop.

File: google/cloud/spanner_v1/snapshot.py [hunk 2]
```diff
@@ -178,12 +178,13 @@ def execute_sql(
             required if parameters are passed.
 
         :type query_mode:
-            :class:`google.cloud.spanner_v1.proto.ExecuteSqlRequest.QueryMode`
-        :param query_mode: Mode governing return of results / query plan. See
-            https://cloud.google.com/spanner/reference/rpc/google.spanner.v1#google.spanner.v1.ExecuteSqlRequest.QueryMode1
+            :class:`~google.cloud.spanner_v1.proto.ExecuteSqlRequest.QueryMode`
+        :param query_mode: Mode governing return of results / query plan.
+            See :
```
Comment (larkee) [inline google/cloud/spanner_v1/snapshot.py:183]: Remove the space before the colon.

**Comment by HemangChothani** [comment]
@larkee I have tried but not able to generate link for proto documentation. I also found similar in [bigtable](https://google-cloud-python.readthedocs.io/en/0.32.0/bigtable/row-filters.html#google.cloud.bigtable.row_filters.ApplyLabelFilter.to_pb) as well where return type is class but unable to generate the link.

**Comment by larkee** [comment]
Ah. I didn't think to check the other clients. In that case, this way is fine.

However, I noticed the issue exists in the client API page as well.

Parameters is split over two lines

It's not as significant but it should a similar change. Other than that, LGTM :+1:

**Comment by HemangChothani** [comment]
For me works fine with the 'firefox' updated version but found issue with the `chrome`.
``projects/<project>/instanceConfigs/<config>`` line creates the problem, to solve this either i have to add space so its look like  ``projects/<project>/instanceConfigs/ <config>`` or  break the line which looks like  ``projects/<project>/instanceConfigs/`` ``<config>``. 
Please recommend.

**Comment by larkee** [comment]
> For me works fine with the 'firefox' updated version but found issue with the `chrome`.
> `projects/<project>/instanceConfigs/<config>` line creates the problem, to solve this either i have to add space so its look like `projects/<project>/instanceConfigs/ <config>` or break the line which looks like `projects/<project>/instanceConfigs/` `<config>`.
> Please recommend.

I prefer the line break.

File: google/cloud/spanner_v1/transaction.py [hunk 3]
```diff
@@ -83,7 +83,7 @@ def begin(self):
             if the transaction is already begun, committed, or rolled back.
         """
         if self._transaction_id is not None:
-            raise ValueError("Transaction already begun")
+            raise ValueError("Transaction google.cloud.spanner_v1.proto.already begun")
```
Comment (larkee) [inline google/cloud/spanner_v1/transaction.py:86]: Revert this line.

**Pull request merged**